pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -127,9 +128,19 @@ void run() throws Exception {
     @Parameters(commandDescription = "Run a health check against the broker")
     private class HealthcheckCmd extends CliCommand {
 
+        @Parameter(names = "--topic-version", description = "topic name version [V1,V2] V1 is default")
+        private String topicVersion;
+
         @Override
         void run() throws Exception {
-            getAdmin().brokers().healthcheck();
+            if (Objects.isNull(topicVersion) || topicVersion.equalsIgnoreCase("V1")) {
+                getAdmin().brokers().healthcheck("V1");
```

Comment:
if we take enum as an input argument then we don't have to validate. we can just call 
`getAdmin().brokers().healthcheck(topicVersion);`